### ISSUE TITLE
Test on an image

### ISSUE THREAD

**Issue by HeshamAMH** [zengqunzhao/MA-Net#1]
Excellent work. But please, I am new in this field. So I wonder if I can use **main.py** to test facial expression on any face from other datasets.
Thanks in advance.

**Comment by zengqunzhao** [comment]
> Excellent work. But please, I am new in this field. So I wonder if I can use **main.py** to test facial expression on any face from other datasets.
> Thanks in advance.

Thank you for your attention to our work. If you want to test on any images, you need to train the model first. Cause currently, the provided codes and pre-trained models are just for training.

**Comment by HeshamAMH** [comment]
Deeply thanks, I trained your model and I obtained accuracy more than 88 % on RAF-DB dataset nearly equal to what you said in the paper. Please, Now I have the model called **[07-29]-[20-53]-model_best.pth** , but as I told you before I am a beginner of using Pytorch. So, is there any code to test this model on any face image.
Thanks again because I took a lot from you precious time.

**Comment by zengqunzhao** [comment]
> Deeply thanks, I trained your model and I obtained accuracy more than 88 % on RAF-DB dataset nearly equal to what you said in the paper. Please, Now I have the model called **[07-29]-[20-53]-model_best.pth** , but as I told you before I am a beginner of using Pytorch. So, is there any code to test this model on any face image.
> Thanks again because I took a lot from you precious time.

I think it's easy to write a code for testing on any image. I  don't have that code now, I am sorry for that.

**Comment by zengqunzhao** [comment]
> > Deeply thanks, I trained your model and I obtained accuracy more than 88 % on RAF-DB dataset nearly equal to what you said in the paper. Please, Now I have the model called **[07-29]-[20-53]-model_best.pth** , but as I told you before I am a beginner of using Pytorch. So, is there any code to test this model on any face image.
> > Thanks again because I took a lot from you precious time.
> 
> I think it's easy to write a code for testing on any image. I don't have that code now, I am sorry for that.

Maybe I can send you that codes, but it may later (two or three mouths) :)

**Comment by HeshamAMH** [comment]
Thanks, I will try.
